packages/block-library/src/home-link/index.php

```diff
@@ -99,12 +99,12 @@ function block_core_home_link_build_li_wrapper_attributes( $context ) {
 		$font_sizes['css_classes']
 	);
 	$style_attribute = ( $colors['inline_styles'] . $font_sizes['inline_styles'] );
-	$css_classes     = trim( implode( ' ', $classes ) ) . ' wp-block-navigation-item';
+	$css_classes     = trim( implode( ' ', $classes ) . ' wp-block-navigation-item' );
 
 	$wrapper_attributes = get_block_wrapper_attributes(
```

Comment:
`get_block_wrapper_attributes` escapes each attribute. I added escaping in here because in the future we might switch to another method which doesn't escape.

Reply:
Possibly. But still you can't guarantee that. It's effectively 🤞 that the developer remembers to do escaping in the new function.

Escaping as near to the point of output as possible is the most robust option even if it introduces additional overhead.

Reply:
>Escaping as near to the point of output as possible is the most robust option 

I agree with that, but I don't think we should add code because it might be needed in the future.

Reply:
Looks like I already removed this.

After some checking I've decided that all usages of `get_block_wrapper_attributes` can be considered safe without having to pre-escape the attributes.